server/channels/app/plugin.go

```diff
@@ -334,6 +334,11 @@ func (ch *Channels) syncPlugins() *model.AppError {
 		return appErr
 	}
 
+	if len(pluginSignaturePathMap) == 0 {
```

Comment:
Semantically, this is unnecessary -- the for loop below is a no-op when the slice is empty -- but logging explicitly here for additional clarity.